Dalamud.Boot/dllmain.cpp

```diff
@@ -5,12 +5,35 @@
 #include "utils.h"
 #include "veh.h"
 #include "xivfixes.h"
+//
+// #include <d3d11.h>
+//
+// #include "hooks.h"
 
 HMODULE g_hModule;
 HINSTANCE g_hGameInstance = GetModuleHandleW(nullptr);
 
 DWORD WINAPI InitializeImpl(LPVOID lpParam, HANDLE hMainThreadContinue) {
     g_startInfo.from_envvars();
+    //
+    // static std::optional<hooks::import_hook<decltype(D3D11CreateDevice)>> s_hookD3D11CreateDevice;
```

Comment:
What is this for? If it's valuable to leave around for someone that might need it in the future, could you leave a comment explaining it?

Reply:
That actually shouldn't be left there, as that's there only for testing. If DirectX debug mode is desired, I think it'd be better to implement that on C# side too?